paasta_tools/api/client.py

```diff
@@ -59,4 +59,10 @@ def get_paasta_api_client(cluster=None, system_paasta_config=None):
         spec_dict = json.load(f)
     # replace localhost in swagger.json with actual api server
     spec_dict['host'] = api_server
-    return SwaggerClient.from_spec(spec_dict=spec_dict)
+
+    # sometimes we want the status code
+    if http_res:
+        config = {'also_return_response': True}
```

Comment:
Why not just do this all the time?

Reply:
If I do this all the time I have to update any call to the client that's already being made to accept a two-ple. This seemed easier, but I can go through and do that.